schema/deploy/tables/cif_attachment.sql

```diff
@@ -0,0 +1,42 @@
+-- Deploy cif:tables/cif_attachment to pg
+
+BEGIN;
+
+create table cif.cif_attachment
+(
+  id integer primary key generated always as identity,
+  file uuid,
+  file_name varchar(1000),
+  s3_path varchar(1000)
```

Comment:
can the `s3_path` be inferred from the `uuid`, if so we don't need a column here. We can add a [computed column](https://www.graphile.org/postgraphile/computed-columns/) later to have an easily-accessible download link (but I think downloading should be a separate PR)

Reply:
you are correct, I removed this field from the schema.